ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1630,6 +1630,13 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-workers-marketplace
+- as: azure-ipi-baselinecaps-additionalcaps-f28
```

Comment:
Can we use this job to replace the existing v4xx ci jobs?

Reply:
no, there are different cases.
1. bsaelineCaps: v4.x/vCurrent
2. baselineCaps: v4.x/vCurrent  + additionalEnabledCapabilites, in this case, there might be some duplicated caps(in both baseline cap set and additionalEnabledCapabilites), to make sure those caps can be handled correctly.

Reply:
> baselineCaps: v4.x/vCurrent + additionalEnabledCapabilites, in this case, there might be some duplicated caps(in both baseline cap set and additionalEnabledCapabilites), to make sure those caps can be handled correctly.

We had ever found some issues before in such case, right?

And one more idea is we can add one more jobs like `<platform>-ipi-baselinecaps-f28` via a random selection to replace the existing v4xx ci jobs, so that we do not have to add one jobs once new version come out.

Reply:
Didn't find issues in such combination "baselineCaps: v4.x/vCurrent + additionalEnabledCapabilit".

Good idea to use one job to cover baselineCaps:v4.x testing. We only found issue on old cap set when introducing new cap so far, and issue happens on all old cap set, for example, "MachineAPI" in 4.14.

let me try to update.